src/appleseed/renderer/modeling/scene/assembly.cpp

```diff
@@ -56,6 +56,8 @@ namespace renderer
 // Assembly class implementation.
 //
 
+const char* Model = "assembly";
```

Comment:
I was following the convention of using the class name for the model, but I think generic_assembly is ok. I'll change it.